src/js/footer/helper/index.js

```diff
@@ -160,8 +162,16 @@ const getDomainAppID = () => {
   }
 };
 
+export const getServerUrl = () => {
+  try {
+    return localStorage.getItem("config.server_url") || DEFAULT_SERVER_URL;
+  } catch (error) {
```

Comment:
I don't think it will reach to catch block if we are returning the default value in any case. 
can't we use like this without try-catch?
`return localStorage.getItem("config.server_url") || DEFAULT_SERVER_URL;`

Reply:
the issue is not on the value taking. it's trackJs complaining on `Failed to read the 'localStorage' property`

Reply:
the issue is 
```
[Trackjs] Failed to read the 'localStorage' property from 'Window': Access is denied for this document
```

it means that the localStorage itself is missing itself, can we just use `if` condition for this? 

```
!!(window?.localStorage)
```

Reply:
yes @prince-deriv can. both approach serve similar purpose in this context. `if` condition explicitly checks `localStorage` is available while using `try-block` will catch any errors during the process.
Updated to use `if` condition.